### PR TITLE
Add Operations API in DevTestLab swagger

### PR DESCRIPTION
**Pull request by ajay-arora**
Add Operations API in DevTestLab 2016 swagger

### CODE REVIEW

File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/2016-05-15/DTL.json [hunk 1]
```diff
@@ -1,8584 +1,8658 @@
...
-                        "in": "path",
-                        "description": "The name of the lab.",
-                        "required": true,
-                        "type": "string"
-                    },
-                    {
-                        "name": "name",
-                        "in": "path",
-                        "description": "The name of the notificationChannel.",
-
```
Comment (anuchandy) [inline specification/devtestlabs/resource-manager/Microsoft.DevTestLab/2016-05-15/DTL.json]: It seems`Operations_List` is missing?

**Review by anuchandy — CHANGES_REQUESTED** [review]

File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/2016-05-15/DTL.json (hunk 1 above)
Comment (anuchandy) [reply specification/devtestlabs/resource-manager/Microsoft.DevTestLab/2016-05-15/DTL.json]: corrosponing [linter error](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/285852306#L575)

Comment (anuchandy) [reply specification/devtestlabs/resource-manager/Microsoft.DevTestLab/2016-05-15/DTL.json]: Talked to @ajay-arora about this today, he clarified that this end-point has nothing to with the `ARM Operations API` that returns all the operations supported by the RP.

The endpoint `/subscriptions/{subscriptionId}/providers/Microsoft.DevTestLab/locations/{locationName}/operations/{name}` is a generic endpoint to query the status of any long running operations in the dev-test lab RP. @ajay-arora  there are bunch of operations marked as `x-ms-long-running-operation,` i guess they return `Azure-AsyncOperation` header with url to poll the status. Based on our discussion format of this header url is `/subscriptions/{subscriptionId}/providers/Microsoft.DevTestLab/locations/{locationName}/operations/{name}`. Right?

Comment (ajay-arora) [reply specification/devtestlabs/resource-manager/Microsoft.DevTestLab/2016-05-15/DTL.json]: Correct

**Review by anuchandy — APPROVED** [review]

**Pull request merged**